vsphere/datadog_checks/vsphere/api.py

```diff
@@ -439,9 +440,12 @@ def get_vsan_metrics(self, cluster_nested_elts, entity_ref_ids, id_to_tags, star
                 )
             health_metrics.append(processed_health_metrics)
 
+            redapl_cluster_data = {'name': cluster_reference.name, 'vcenter_server': self.config.hostname}
```

Comment:
Do we always want to send this data when `collect_vsan_data` is set to `True`? Could we document this behavior as well in the conf.yaml and readme?

Reply:
Yes, whenever this vsan resource is reported as part of the vsphere check process, it's associated resource metadata should also be sent. I will update the documentation accordingly.